package/src/components/Message/Message.tsx

```diff
@@ -312,7 +313,9 @@ const MessageWithContext = <
     goToMessage(quotedMessage.id);
   };
 
-  const onPress = (error = message.type === 'error' || message.status === 'failed') => {
+  const onPress = (
+    error = message.type === 'error' || message.status === MessageStatusTypes.FAILED,
+  ) => {
```

Comment:
```suggestion
  const messageError = message.type === 'error' || message.status === MessageStatusTypes.FAILED;
  const onPress = (
    error = messageError
  ) => {
```

Not really so much a suggestion as me just moving the check out so my brain would parse it easier. 

This check happens in a couple of other places as well, should we extract it to some kind of helper?